### ISSUE TITLE
Improve members attributes endpoint

### ISSUE THREAD

**Issue by joanagmaia** [CrowdDotDev/crowd.dev#1586]
### Problem

Currently, the api/tenant/{id}/settings/members/attributes/{id} endpoint is returning all available values for an attribute. For instance, in the Linux environment, the skills attribute is returning 9k results which makes the request really slow and degrades performance of the pages

### Possible solution
Paginate endpoint so that frontend can request x results at once
This will require frontend and backend

**Comment by IAmRiteshKoushik** [comment]
Hey @joanagmaia , I would like to work on resolving this issue. Can I have this assigned to me? Also, where do I direct my queries and discussions for resolving this issue ?

**Comment by techmannih** [comment]
hello @joanagmaia ,
I want to work on this issue....... please assign me .

**Comment by joanagmaia** [comment]
@IAmRiteshKoushik are you working on this?

**Comment by IAmRiteshKoushik** [comment]
Yes, I am still working in this. Will send a PR within a week. Please bear with me.

**Comment by joanagmaia** [comment]
> Yes, I am still working in this. Will send a PR within a week. Please bear with me.

No problem! Just checking if you still wanted to contribute on this one 😄
